src/04-modality-specific-files/05-task-events.md

```diff
@@ -86,6 +108,33 @@ onset duration  trial_type  identifier  database  response_time
 5.6 0.6 sad AF01ANSA  kdef  1.739
 ```
 
+This file should be accompanied by a data dictionary.  
+
+```Text
+sub-control01/
+    func/
+        sub-control01_task-emoface_events.json
+```
+
+```JSON  
+{
+   "trial_type": {
+       "LongName":   "Emotion image type",
+       "Descripton": "Type of emotional face from Karolinska database that is displayed",
+       "Levels": {
+          "afraid": "A face showing fear is displayed",
+          "angry":  "A face showing anger is displayed",
+		  "sad":    "A face showing sadness is displayed"
+        }
+   },
+   "identifier": {
+       "LongName": "Unique identifier from Karolinska (KDEF) database",
+       "Description": "ID from KDEF database used to identify the displayed image"
+   }
+}
```

Comment:
same comment here as above: we need to consider adding JSON descriptions for all columns (or not)

onset duration  trial_type  identifier  database  response_time

Reply:
Same comment.  Whole events specification, examples, and validation needs some work.  The current pull request is needed to move forward on this.  I think we should separate it and tackle incrementally.